.github/workflows/vuln-scans.yaml

```diff
@@ -0,0 +1,41 @@
+name: Vulnerability scans
+on:
+  workflow_dispatch: {}   # Allow for manual triggers
+  schedule:
+    - cron:  '0 8 * * *'  # Daily, at 8:00 UTC
+  pull_request:
+
+jobs:
+  trivy-scan:
+    name: Trivy security scan
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Run Trivy scan on repo
+        uses: aquasecurity/trivy-action@master
```

Comment:
Are there any versions for this? Maybe `master` is a bit too bleeding-edge? 😉

Reply:
Does OPA have automatic updates for GH actions?

Reply:
[Yes](https://github.com/open-policy-agent/opa/blob/main/.github/dependabot.yml#L7)